index.html

```diff
@@ -157,69 +161,144 @@ <h3>Apoie este projeto</h3>
           </p>
         </div>
         <div class="wrapper-cards-modal">
-          <div class="card-modal">
-            <input type="radio" />
-            <div class="card-without-reward">
-              <h4>Apoio sem recompensa</h4>
-              <p class="info-description-modal">
-                Escolha nos apoiar sem recompensa se você simplesmente acreditar
-                em nosso projeto. Como apoiador, você será assinado up para
-                receber atualizações de produtos por e-mail.
-              </p>
+          <div class="wrapper-card">
```

Comment:
uma boa pratica é começar o nome igual para o principal e todos os filhos que se aplicam, aqui por exemplo o wrapper se chamar `card-wrapper` e o fliho `card-modal` pq quando vc for usar algo como um sass ou syled components vc vai reaproveitar o nome, de novo, só algo pra vc olhar pro futuro, n se estressa com esse aqui.

Reply:
Certo, obrigado